### DISCUSSION TITLE
Send email verifiy via action

### DISCUSSION THREAD

**Discussion by xBigDaddyx** [filamentphp/filament#13848] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I have user resource for admin to manage their users, i want to send email verification by action on table. but i can find any documentation on filament page?

**Comment by leandrocfe** [comment, marked as answer]
You need to create a [custom laravel notification](https://medium.com/@francescopassanante/how-send-an-email-notification-to-new-users-on-laravel-filament-application-c44eaa67d716)

**Reply by xBigDaddyx** [reply]
thank you.
